services/ui-src/src/utils/other/time.ts

```diff
@@ -141,6 +159,16 @@ export const checkDateRangeStatus = (
   return currentTime >= startDate && currentTime <= endDate;
 };
 
+export const calculatePeriod = (dueDate: string) => {
+  const dueDateAsDate = new Date(dueDate);
+  if (
+    dueDateAsDate >= beginningPeriodOne(dueDate) &&
+    dueDateAsDate <= endPeriodOne(dueDate)
+  )
+    return 1;
+  return 2;
```

Comment:
are these used for anything other than being part of the report name? If not I'd consider returning them as a string rather than an integer, just so that we don't have any confusion with these values being used for calculations and/or 1=true, 2=? kind of thing

Reply:
i don't believe they are used for anything else, but I will double check with product. I think making them strings is a good idea, regardless! will revise :)